src/index.html

```diff
@@ -20,7 +20,7 @@ <h1>Excepteur Sint Occaecat</h1>
         </a>
       </li>
     </ul>
-    <h2 class="highlight point">Duis Aute</h2>
+    <h2 class="point">Duis Aute</h2>
```

Comment:
Does the class `highlight` get used by other elements?

If not, we can just remove it, since nothing's using it.

Reply:
It wasnt being used, so I removed it. Thanks!